cfy_manager/logger.py

```diff
@@ -107,9 +107,13 @@ def _setup_logger():
     _setup_file_logger(logger)
 
 
-def setup_console_logger(verbose=False, init=[]):
-    if not init:
-        init.append(True)
+logging_initialized = False
+
+
+def setup_console_logger(verbose=False):
+    global logging_initialized
```

Comment:
Not overly happy with a global, but also:
Shouldn't the logging_initialized = True be set after initializing it, rather than before?
(and the global should be declared only just before we modify it)

Reply:
@geokala sure, can be after. This was just changing the implicit global to an explicit one. Updated